.github/workflows/release-drafter.yml

```diff
@@ -28,12 +28,12 @@ jobs:
         # Preface Traffic Capture version with 0. to signal interface immaturity
         run: |
           wget https://github.com/opensearch-project/opensearch-migrations/archive/refs/tags/${{ steps.get_data.outputs.version }}.tar.gz -O artifacts.tar.gz
-          (cd TrafficCapture && ../gradlew publishMavenJavaPublicationToMavenRepository -Dbuild.snapshot=false -Dbuild.version=0.${{ steps.get_data.outputs.version }} && tar -C build -cvf traffic-capture-artifacts.tar.gz repository)
+          ./gradlew publishMavenJavaPublicationToMavenRepository -Dbuild.snapshot=false -Dbuild.version=0.${{ steps.get_data.outputs.version }} && tar -C build -cvf traffic-capture-artifacts.tar.gz repository
       - name: Draft a release
         uses: softprops/action-gh-release@v2
         with:
```

Comment:
What do you think about adding `fail_on_unmatched_files` option from https://github.com/softprops/action-gh-release?

Reply:
I think that would be a good addition. We did fail this action somehow, but it seems there may be some edge cases that the flag can help with